### PR TITLE
LPS-139938 / LPS-145450 Implement Azure translator service

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

File: modules/apps/translation/translation-google-cloud-translator/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,6 @@
-Bundle-Name: Liferay Translation Google Cloud Translator
-Bundle-SymbolicName: com.liferay.translation.google.cloud.translator
-Bundle-Version: 1.0.11
+Bundle-Name: Liferay Translation Translator Google Cloud
+Bundle-SymbolicName: com.liferay.translation.translator.google.cloud
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-google-cloud-translator/bnd.bnd:2]: looking at the failling poshi tests there is calls to the old module `com.liferay.translation.google.cloud.translator` (as on configuration on DMNavigator.macro) and other calls that maybe must be changed.

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/translation/translation-google-cloud-translator/bnd.bnd (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/translation/translation-google-cloud-translator/bnd.bnd:2]: dupe!

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-google-cloud-translator/bnd.bnd (hunk 1 above)
Comment (adolfopa) [reply modules/apps/translation/translation-google-cloud-translator/bnd.bnd:2]: Done.

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**